src/server/licensing_func.c

```diff
@@ -286,8 +286,13 @@ propagate_licenses_to_vnodes(mominfo_t *pmom)
 	if (node_index_start)
 		distribute_licenseinfo(pmom, lic_count);
 
-	if (pfrom_Lic == NULL)
+	if (pfrom_Lic == NULL) {
```

Comment:
why need a different loop for this? Couldn't you accommodate this with the previous loop without a break.

Reply:
done!